emoji-kotlin/src/main/kotlin/com/vanniktech/emoji/kotlin/EmojiExtensions.kt

```diff
@@ -0,0 +1,7 @@
+package com.vanniktech.emoji.kotlin
+
+import com.vanniktech.emoji.EmojiUtils
+
+fun String.isOnlyEmojis() = EmojiUtils.isOnlyEmojis(this)
+
+fun String.emojisCount() = EmojiUtils.emojisCount(this)
```

Comment:
I think one more extension method is now missing since I added one more to EmojiUtils